src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -447,6 +471,17 @@ protected function initializeBundles()
     }
 
     /**
+     * The extension point similar to the Bundle::build() method.
+     *
+     * Use this method to register compiler passes and manipulate the container during the building process.
+     *
+     * @param ContainerBuilder $container
+     */
+    protected function build(ContainerBuilder $container)
```

Comment:
So isnt `Kernel::buildContainer` the method `Bundle::build` tends to be, ie.. do we still need that?

If you go bundles, you probably (should) go extensions anyway.